### PR TITLE
Enrich the README

### PR DESCRIPTION
**Pull request by jotanavarro**
I updated the `README.md` file to add more information about the installation process on other operating systems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
 
 You can find a description of the [available commands](./docs/hmctl.md) in this repository.
 
-## Mac OS Installation Instructions
+## Installation
+
+### GNU/Linux
```
Comment (misterikkit) [inline README.md:9]: Mention of GNU seems irrelevant here.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,17 @@
...
+
+In GNU/Linux, you can simply call the binary from the command line, there are no special requirements.
```
Comment (misterikkit) [inline README.md:11]: Because of the way we currently package binaries on the release page, users will need to `chmod +x` the file before they can run it. It also should be renamed to just `hmctl` and placed somewhere in the user's `$PATH`.

I am not sure what level of detail we need to go into here. And we definitely should improve how we package and ship these binaries (separate effort).

File: README.md [hunk 3]
```diff
@@ -4,6 +4,17 @@
...
+
+### MacOS
+
+For MacOS, you simple have to right-click the file and select "Open".  Your system will warn you that the file cannot be opened because it is from an _unidentified developer_.  That is fine.  You have to go to _System Preferences_ -> _Security & Privacy_ -> _General_ and then click to "Open Anyway".
+
+### Windows
+
+You have to rename the binary to add the `.exe` extension.  Then you can open it in your terminal as usual.
```
Comment (misterikkit) [inline README.md:19]: Ugh I think that's a bug in the release script. It should have been uploaded as a .exe.

File: README.md [hunk 4]
```diff
@@ -4,6 +4,17 @@
...
-## Mac OS Installation Instructions
+## Installation
+
+### GNU/Linux
+
+In GNU/Linux, you can simply call the binary from the command line, there are no special requirements.
+
+### MacOS
+
+For MacOS, you simple have to right-click the file and select "Open".  Your system will warn you that the file cannot be opened because it is from an _unidentified developer_.  That is fine.  You have to go to _System Preferences_ -> _Security & Privacy_ -> _General_ and then click to "Open Anyway".
```
Comment (misterikkit) [inline README.md:15]: typo: s/simple/simply/

Comment (misterikkit) [inline README.md:15]: Which version of MacOS requires this workflow? I think whichever version we tested on originally worked with the instructions we had before. 😕

File: README.md (hunk 2 above)
Comment (jotanavarro) [reply README.md:11]: Oh true!  I forgot about that and it was in my list of to-dos.  Thank you for noticing!

I think the `$PATH` is a little bit beyond the scope of what I was thinking about originally.

File: README.md (hunk 3 above)
Comment (jotanavarro) [reply README.md:19]: Then should I send this change or just leave it as-is until it is fixed?

File: README.md (hunk 4 above)
Comment (jotanavarro) [reply README.md:15]: I saw this issue today on Monterrey, running on Apple silicon.  Maybe you were executing your locally build binary?

File: README.md (hunk 3 above)
Comment (misterikkit) [reply README.md:19]: Let's change this now and change it again when it's fixed.

**Review by misterikkit — APPROVED** [review]

**Review by purestorage-jorson — APPROVED** [review]

**Review by purestorage-fusion — APPROVED** [review]
